Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -1293,6 +1295,25 @@ int CLuaVehicleDefs::GetHelicopterRotorSpeed(lua_State* luaVM)
     return 1;
 }
 
+std::variant<bool, float> CLuaVehicleDefs::GetVehicleRotorSpeed(CClientVehicle* pVehicle)
+{
+    switch (pVehicle->GetVehicleType())
+    {
+        case CLIENTVEHICLE_PLANE:
+        {
+            return pVehicle->GetPlaneRotorSpeed();
+        }
+        case CLIENTVEHICLE_HELI:
+        {
+            return pVehicle->GetHeliRotorSpeed();
+        }
```

Comment:
Wouldn't it be easier to read this, if you moved this logic to `CClientVehicle`?
For example `bool CClientVehicle::GetRotorSpeed(float&)` and `bool CClientVehicle::SetRotorSpeed(float)`

Reply:
updated :)